ros2-library/src/test/java/us/ihmc/ros2/CommunicationTest.java

```diff
@@ -46,9 +44,8 @@ private void testSimpleCommunication()
          Pair<Integer, Integer> messagesReceived = new MutablePair<>();
          try
          {
-            Domain domain = DomainFactory.getDomain();
             String name = "ROS2CommunicationTest";
-            ROS2Node node = new ROS2Node(domain, name);
+            ROS2Node node = new ROS2NodeBuilder().build(name);
```

Comment:
What if build was a static method on ROS2Node, so you could do `ROS2Node.build(name).withSettings().etc()..`